src/components/schema/jsonld.tsx

```diff
@@ -0,0 +1,92 @@
+import {LocalBusiness, Thing, WithContext, PostalAddress} from 'schema-dts';
+import PhotoGallery from '../photo-gallery';
+
+export function JsonLd<T extends Thing>(json: WithContext<T>): string {
+  return `<script type="application/ld+json">
+${JSON.stringify(json)}
+</script>`;
+}
+
+export const SchemaWrapper = (data: any) => {
```

Comment:
I wonder if we should be defining the Streams request in typescript (since we have to declare the fields to return anyways). If so we could leverage it here
```suggestion
export const SchemaWrapper = (data: StreamsResponse) => {
```
This is out of scope of what you're doing, just wanted to get the thought down